apps/builder/app/canvas/canvas.tsx

```diff
@@ -167,7 +167,7 @@ export const Canvas = ({ params }: CanvasProps): JSX.Element | null => {
   const components = new Map(
     Object.entries({ ...baseComponents, ...remixComponents })
   ) as Components;
-  useSyncInitializeOnce(() => {
+  useMount(() => {
```

Comment:
this is basically useEffect(effect, []);

Reply:
useMount useUnmount is std hooks in react-use 
they are more clear than `useEffect(() => {}, [])`
They do not interfere with eslint i.e. add dependency x into `[]`